### ISSUE TITLE
Error: Directory nonexistent. Code 512

### ISSUE THREAD

**Issue by RpfR2000** [barricklab/breseq#350]
Hi -- first off, thanks for making this software publicly available! It is very cool.

I'm getting the following error: 

sh: 1: cannot create <my-dir>/output/evidence/512.r.log: Directory nonexistent
Creating coverage plot for region: contig_36:1-2780
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!> FATAL ERROR <!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Error running command:
[system] R --vanilla < "<path-to-breseq>/plot_coverage.r" > "<my-dir>/output/evidence/512.r.log" --args in_file="<my-dir>/output/evidence/512_1.coverage.tab" out_file="<my-dir>/output/evidence/contig_1.overview.png" pdf_output=0 total_only=0 window_start=1 window_end=738339 avg_coverage=0.0 fixed_coverage_scale=0
Result code: 512
FILE: libbreseq/common.h   LINE: 1411

This error occurred during mutation annotation in polymorphism mode (-p, -o, -j were the only flags I used in the original command). I tried running the command that is causing the error a as a standalone and it worked fine, which might mean it is an issue with my installation of breseq. I'm using bowtie2 version 2.4.5, R version 4.2.3, and breseq version 0.38.1. 

On a side note, I am getting a few warnings. Should I be concerned about these? They only apply to a few contigs. I'm afraid I'm not informed enough about genomics to know whether these are a problem:

Failed to fit coverage distribution for some reference sequences. This may degrade the quality of predicting mutations from new sequence junctions (JC evidence). You may want to set --deletion-coverage-propagation-cutoff to improve the quality of deletion prediction (MC evidence).

Insufficient coverage to call mutations for some reference sequences. Set either the --targeted-sequencing or --contig-reference option if you want mutations called on these reference sequences.

Thanks!

**Comment by jeffreybarrick** [comment]
I can think of a few possibilities for the error...
1) You ran out of hard drive space in the middle of the _breseq_ run.
2) You ran _breseq_ in different ways as users with different permissions on the same output folder.

Probably, it would be best to delete all of the output and start again from the beginning.

To make sure your environment is consistent, you might also try installing _breseq_ through Bioconda.

The errors about failing to fit the coverage distribution and having low coverage seem to indicate that you either don't have enough reads for _breseq_ to call mutations, or you are using a reference sequence that is not sufficiently similar to the genome of the microbe you sequenced for reads to be mapped.

**Comment by jeffreybarrick** [comment]
That's a really big discrepancy...I think it points to something being unexpected about the data–probably that is is very divergent from the reference genome. 

Is there good coverage of the reference genome when you look at the summary output for consensus mode?

**Comment by RpfR2000** [comment]
Okay, so the timing issue was entirely due to the output directory. When I ran in consensus mode, I let the output be the root directory, whereas for some reason I set the output directory to a data drive when using polymorphism mode. They're roughly equivalent in terms of computation time when outputting to the same directory. Coverage is really good (>95%) so that wasn't the issue. Sorry for the confusion!
